datastore/data_quality/quality_data.py

```diff
@@ -357,12 +363,8 @@ def get_publisher_quality_grants(self):
                 continue
 
             # For compatibility badge value 100 = True , 0 = False
-            ret[metric] = (
-                100
-                if self.source_file_set.count()
-                == self.source_file_set.filter(**query).count()
-                else 0
-            )
+            ret[metric] = 100 if self.source_file_set.filter(**query).count() > 0 else 0
```

Comment:
Am I understanding this right: The metric changed from "all source files must match the condition" to "at least one source file must match the condition"?

Reply:
Yes that's correct.

Reply:
Now that you've reminded me of that I think that could do with being in the commit message.

Reply:
Updated commit message